example_configs/ignite_2.6.0.yml

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+#
+#
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+rules:
+# Apache Ignite provide runtime information on a cluster.  Keep in mind that there will be a certain
```

Comment:
It'd be better to get the data from the individual processes directly, rather than adding jitter and lag.

Reply:
Brian, Could you, please, calrify? In general, I agree that it's better to share individual processes metrics. But I would like to have an batch of heap and non-heap metrics for the whole cluster.